aries-site/src/components/content/BulletedList.js

```diff
@@ -1,19 +1,32 @@
 import React from 'react';
 import PropTypes from 'prop-types';
-import { Text } from 'grommet';
+import { List, Text } from 'grommet';
 
-export const BulletedList = ({ items }) => {
+import { TEXT_SIZE } from '../../layouts';
+
+export const BulletedList = ({ items, level, size }) => {
   return (
-    <>
-      {items.map(item => (
-        <Text margin={{ bottom: 'xsmall' }} key={item}>
+    <List
+      data={items}
+      border={{ style: 'hidden' }}
+      pad={{ vertical: 'xxsmall' }}
+    >
+      {item => (
+        <Text key={item} size={size || TEXT_SIZE[level]}>
           — {item}
```

Comment:
Looks like designs call for a smaller dash -- is that possible?

Reply:
fixed by deleting then typing the hyphen, must've been a copy-paste which charachter encoded differently